rook/doc/rook.tex

```diff
@@ -0,0 +1,81 @@
+\documentclass{article}
+
+\title{Rook Test Description}
+
+\begin{document}
+\maketitle
+
+\section{Testers}
+
+The testers are used to run a test.  The tester can fail the test, or
+it can be failed by one of the differs that are attached as subnodes
+in the test input.
+
+\subsection{Common Features}
+
+These are the features that are common to all testers.
+
+\begin{description}
+  \item[type] The type of this test
```

Comment:
Could we add a little to this description to indicate what a "type" is? Is this like "normal, heavy" or like "RavenFramework, Python"?

Reply:
Added examples to run_types and type.